app/assets/javascripts/components/bulk-edit-measures-save-actions.js.coffee

```diff
@@ -127,37 +107,33 @@ window.BulkEditOfMeasuresSaveActions =
 
     exit_link.addClass('disabled')
 
-  unlock_buttons: ->
+  unlockButtons: ->
     $(".js-bulk-edit-of-measures-save-progress").removeClass('disabled')
     $(".js-bulk-edit-of-measures-submit-for-cross-check").removeClass('disabled')
     $(".js-bulk-edit-of-measures-exit").removeClass('disabled')
 
-  closeErrorDetailsPopup: ->
-    $(document).on 'click', '.js-bulk-edit-of-measures-error-details-close-popup', ->
-      MicroModal.close("modal-1530613430")
-      $(".js-bulk-edit-of-measures-error-details").html("")
+  showSummaryPopup: ->
+    modal_id = "bem-save-progress-summary"
+    content = "There are no any validation errors!"
 
-      return false
+    if $(".has-validation-errors").length > 0
+      content = "Some measures are having validation errors! <br /> Please review table cells with highighted with red."
```

Comment:
Some measures have conformance errors, please review table cells with highlighted in red